cpp/examples/ide_secir.cpp

```diff
@@ -26,7 +26,6 @@
 #include "memilio/math/eigen.h"
 #include "memilio/utils/time_series.h"
 #include "memilio/epidemiology/uncertain_matrix.h"
```

Comment:
```suggestion
```

This include is superfluous. Check for others

Reply:
I am using an UncertainContactMatrix in line 86. Why is this superflous?